tests/driver_at86rf215/main.c

```diff
@@ -74,8 +77,125 @@ static int cmd_enable_batmon(int argc, char **argv)
     return res;
 }
 
+static int cmd_set_trim(int argc, char **argv)
+{
+    if (argc < 2) {
+        printf("usage: %s <trim>\n", argv[0]);
+        return 1;
+    }
+
+    uint8_t trim = atoi(argv[1]);
+
+    if (trim > 0xF) {
+        puts("Trim value out of range");
+        return 1;
+    }
+
+    gnrc_netif_t *netif = gnrc_netif_get_by_type(NETDEV_AT86RF215, 0);
+
+    if (netif == NULL) {
+        puts("No at86rf215 radio found");
+        return 1;
+    }
+
+    at86rf215_t* dev = (at86rf215_t*)netif->dev;
+
+    printf("setting trim to %u fF\n", 300U * trim);
+    at86rf215_set_trim(dev, trim);
+
+    return 0;
+}
+
+static int cmd_set_clock_out(int argc, char **argv)
+{
+    const char *keys[] = {
+        [AT86RF215_CLKO_OFF]    = "off",
+        [AT86RF215_CLKO_26_MHz] = "26",
+        [AT86RF215_CLKO_32_MHz] = "32",
+        [AT86RF215_CLKO_16_MHz] = "16",
+        [AT86RF215_CLKO_8_MHz]  = "8",
+        [AT86RF215_CLKO_4_MHz]  = "4",
+        [AT86RF215_CLKO_2_MHz]  = "2",
+        [AT86RF215_CLKO_1_MHz]  = "1",
+    };
+
+    at86rf215_clko_freq freq = AT86RF215_CLKO_26_MHz;
+
+    if (argc > 1) {
+        unsigned tmp = 0xFF;
+        for (unsigned i = 0; i < ARRAY_SIZE(keys); ++i) {
+            if (strcmp(argv[1], keys[i]) == 0) {
+                tmp = i;
+                break;
+            }
+        }
+
+        if (tmp == 0xFF) {
+            printf("usage: %s [freq in MHz | off]\n", argv[0]);
+            printf("valid frequencies: off");
+            for (unsigned i = 1; i < ARRAY_SIZE(keys); ++i) {
+                printf(", %s MHz", keys[i]);
+            }
+            puts("");
```

Comment:
When using `atoi()`, you could go with this instead:
```suggestion
            printf("valid frequencies: off, 26 MHz");
            for (unsigned i = 1; i <= 32; i <<= 1) {
                printf(", %u MHz", i);
            }
            puts("");
```